cmd/erasure-server-pool.go

```diff
@@ -1556,6 +1556,17 @@ func (z *erasureServerPools) ListObjects(ctx context.Context, bucket, prefix, ma
 		last := objects[len(objects)-1]
 		loi.NextMarker = opts.encodeMarker(last.Name)
 	}
+
+	if merged.lastSkippedEntry != "" {
+		if merged.lastSkippedEntry > loi.NextMarker {
```

Comment:
```suggestion
		if merged.lastSkippedEntry > loi.NextMarker {
```

isn't loi.NextMarker already encoded here? @vadmeste how will this work?